src/js/edu-benefits/1995/components/IntroductionPage.jsx

```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
 import ProgressButton from '../../../common/components/form-elements/ProgressButton';
 
 class IntroductionPage extends React.Component {
-  constructor() {
-    super();
-    this.goForward = this.goForward.bind(this);
-  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
-  goForward() {
+  goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
+  goBack = () => {
+    // Wasn't able to figure out how to get react-router to go to
+    //  /education/apply-for-education-benefits/ without having it prepend the
+    //  pathname with /education/apply-for-education-benefits/application/
+    // this.props.router.replace('/education/apply-for-education-benefits/');
+    window.location = "/education/apply-for-education-benefits/";
```

Comment:
Sure thing. I didn't want to do that initially because it'll have a slightly different hover behavior (the link location will appear at the bottom of the page), but I can certainly make that happen.